src/compiler/diagnosticMessages.json

```diff
@@ -1447,6 +1447,10 @@
         "category": "Error",
         "code": 2501
     },
+    "Cannot create an instance of the abstract class '{1}'": {

```

Comment:
Oke, Will change this (but I will wait for feedback from JsonFreeman)
The message I got now is more or less the same as the one from C#.